frontend/src/pages/ExploreData/ExploreDataPage.tsx

```diff
@@ -96,8 +93,10 @@ function ExploreDataPage() {
   const onboardingCallback = (data: any) => {
     if ([STATUS.FINISHED, STATUS.SKIPPED].includes(data.status)) {
       setActivelyOnboarding(false);
-      // TODO(kristak): Add cookies back
-      // setCookie("skipOnboarding", true, { path: "/" });
+      const expirationDate = new Date();
+      // Expiration date set for a year from now
+      expirationDate.setFullYear(expirationDate.getFullYear() + 1);
```

Comment:
Should this be a shorter time frame? Maybe let's ask the group in slack

Reply:
Started a slack thread- I'll submit now to avoid merge issues but it's easy to update later if we want to shorten it